### PR TITLE
add .gitattributes file

### LINKED ISSUE #11: Useless files are distributed
After installing this package via composer, I see the following files in `vendor/deplyer/dist` directory:

> composer.json  dep  .gitignore  README.md  update

The only file that we actually need is `dep`.
Other files should be removed (added to "export-ignore" in .gitattributes?)

### PR DESCRIPTION
**Pull request by garak**
Fix #11

### CODE REVIEW

**Comment by garak** [comment]
@antonmedv?

**Comment by antonmedv** [comment]
Hi, sorry for the delay. Forget about this one)

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,4 @@
+/.git*          export-ignore
+/composer.json  export-ignore
```
Comment (antonmedv) [inline .gitattributes:2]: Maybe don't ignore this one?

Comment (garak) [reply .gitattributes:2]: Why not? What can be the purpose of distributing composer.json?

Comment (garak) [reply .gitattributes:2]: Gentle reminder

**Pull request merged**